docs/usage/capabilities_list.md

```diff
@@ -0,0 +1,75 @@
+CapabilitiesList
+========================================
+
+Overview
+========================================
+
+**CapabilitiesList** is a Java application to list the capabilities
+of the "Mozilla-JSS" provider and other installed providers.
+It does it in two fashions brief and verbose. 
+
+It lets JSS contributors see what features the JDK implements (at different versions)
+and see what else we'd need to add in, to reach compatibility. Since we're 
+attempting to be a default crypto provider, it would be nice to ensure we're 
+mostly close to what the JDK provides. Hopefully other packages written to the
+JDK provider can have similar algorithms available under JSS and behave well. 
+Large gaps in coverage are likely an issue (such as #341 and #242).
+
+Usage
+========================================
+First build jss according to the instructions in jss/README.md
+You should see in the build directory tests_jss4.jar which is what
+contains the application along with the regular tests.
+From the jss/build directory execute
+./run_test org.mozilla.jss.tests.Capabilities
```

Comment:
Based on our discussion on https://github.com/dogtagpki/jss/issues/693, I think this command is wrong. I think it should be:

```suggestion

    ./run_test.sh org.mozilla.jss.tests.CapabilitiesList
```

(Note extra space and indentation to turn it into a code block like below) :-)